data/json/recipes/weapon/mods.json

```diff
@@ -872,6 +872,34 @@
     "tools": [ [ [ "soldering_standard", 10, "LIST" ], [ "small_repairkit", 10 ], [ "large_repairkit", 10 ] ] ],
     "components": [ [ [ "scrap", 3 ], [ "steel_chunk", 1 ] ], [ [ "plastic_chunk", 2 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "wooden_grip",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "fabrication",
+    "skills_required": [ [ "gun", 3 ] ],
+    "difficulty": 6,
```

Comment:
> This one was cleverly handmade by a desperate survivor or miserly enthusiast

>Fab 6

🤔 I think you might need to lower this requirement.

Reply:
The existing grip requires fabrication 5 to craft, as this requires some lateral thinking I thought a small increase was appropriate. Are you thinking it should be 5, or something lower than that?

Reply:
Attachment by hammering nails in is much simpler than the screwcap mechanism our grip is stated to be based on([Daniel Defense Tornado, 2.4oz grip](https://danieldefense.com/daniel-defense-vertical-foregrip-daniel-defense-tornado.html), installation tutorial [here](https://youtu.be/8uXSYElIJ20?t=106))
Higher fabrication & installation requirements means this gunmod has no gameplay niche and will never be used outside of DIY aesthetics or self-imposed challenges(ex. Innawoods = no easy access to soldering irons).